### PR TITLE
move icons to assets + add es module support

### PR DESCRIPTION
**Pull request by jonathanadler**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -120,6 +122,7 @@
   },
   "yoshi": {
     "exports": "wix-ui-tpa",
-    "tpaStyle": true
+    "tpaStyle": true,
+    "entry": "./index.js"
```
Comment (shlomitc) [inline package.json]: not sure this is required

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import * as Icons from './assets/icons';
+export { Icons };
```
Comment (shlomitc) [inline src/index.js]: so do you suggest using this or what's in line 1?

Comment (jonathanadler) [reply src/index.js]: It seems that using:
```
export * as Icons from './assets/icons';
```
Is experimental and not supported.
The solution is to split it to two:
```
import * as Icons from './assets/icons';
export { Icons };
```

File: package.json (hunk 1 above)
Comment (jonathanadler) [reply package.json]: I guess that's true

File: src/index.js (hunk 2 above)
Comment (jonathanadler) [reply src/index.js]: @shlomitc was this what you meant?

**Pull request merged**